### ISSUE TITLE
Debug tracer fails on invalid opcodes

### ISSUE THREAD

**Issue by garythung** [paradigmxyz/reth#3254]
### Describe the bug

Attempting to run a debug_traceTransaction on [this failed Goerli tx](https://goerli.etherscan.io/tx/0x9871881260ee88b1a18509aeeba95da45ff0b8ddfcfa1e67031b219c3433fb9f). Another repro case is [this Sepolia tx](https://sepolia.etherscan.io/tx/0xdd167783aee097c5aa331349fbbdf01558f8c0b10ccb55ea0a7cc872bb863a8c).

Get this error on reth:

`thread 'tokio-runtime-worker' panicked at 'is valid opcode;', /home/gary/reth/crates/revm/revm-inspectors/src/tracing/mod.rs:219:18`

Get this return value when calling my node

```
{"jsonrpc":"2.0","error":{"code":-32700,"message":"Parse error"},"id":null}
```

Get this return value when calling an Alchemy node

```
{"jsonrpc":"2.0","id":1,"result":{"from":"0x852482ee9a74ae0c3a7a3d61ce461207411c8327","gas":"0x293e8","gasUsed":"0x36330","input":"0xbcdfe0d5","error":"invalid opcode: opcode 0xbc not defined","value":"0x0","type":"CREATE"}}
```

### Steps to reproduce

```
curl -X POST \
  -H "Content-Type: application/json" \
  --data '{
    "jsonrpc": "2.0",
    "method": "debug_traceTransaction",
    "params": ["0x9871881260ee88b1a18509aeeba95da45ff0b8ddfcfa1e67031b219c3433fb9f", {"tracer": "callTracer", "tracerConfig": {"onlyTopCall": false}}],
    "id": 1
  }' \
  <INSERT GOERLI RPC URL HERE>
```

### Node logs

```text
thread 'tokio-runtime-worker' panicked at 'is valid opcode;', /home/gary/reth/crates/revm/revm-inspectors/src/tracing/mod.rs:219:18
```
```


### Platform(s)

Linux (x86)

### What version/commit are you on?

reth Version: 0.1.0 Commit SHA: 501fd0d9

### What database version are you on?

Current database version: 1
Local database version: 1

### If you've built Reth from source, provide the full command you used

RUSTFLAGS="-C target-cpu=native" cargo build --profile maxperf --features=jemalloc

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by mattsse** [comment]
oh nice find!

we should simply ignore this during tracing because this will be handled by the evm

**Comment by mattsse** [comment]
will be fixed by #3255

but reth's error message should be less helpful, needs some work
